jerry-core/jrt/jrt-bit-fields.h

```diff
@@ -16,7 +17,27 @@
 #ifndef JERRY_BIT_FIELDS_H
 #define JERRY_BIT_FIELDS_H
 
-extern uint64_t __attr_const___ jrt_extract_bit_field (uint64_t, size_t, size_t);
-extern uint64_t __attr_const___ jrt_set_bit_field_value (uint64_t, uint64_t, size_t, size_t);
+/**
+ * Extract a bit-field.
+ *
+ * @param container container to extract bit-field from
+ * @param lsb least significant bit of the value to be extracted
+ * @param width width of the bit-field to be extracted
+ * @return bit-field's value
+ */
+#define jrt_extract_bit_field(container, lsb, width) \
```

Comment:
Use uppercase for name of macros
`#define JRT_EXTRACT_BIT_FIELD(container, lsb, width) \`